test/e2e/framework/nodes_util.go

```diff
@@ -61,14 +61,15 @@ func masterUpgradeGKE(v string) error {
 	return err
 }
 
-var NodeUpgrade = func(f *Framework, v string) error {
+var NodeUpgrade = func(f *Framework, v string, img string) error {
 	// Perform the upgrade.
 	var err error
 	switch TestContext.Provider {
 	case "gce":
+		// TODO(maisem): add GCE support for upgrading to different images.
```

Comment:
Yeah, but it needs some investigation whether it will break anything else. I'll come back to this after 1.4.